dashboard/test/controllers/survey_results_controller_test.rb

```diff
@@ -20,4 +20,33 @@ class SurveyResultsControllerTest < ActionController::TestCase
     assert_equal 3, survey_result["properties"]["survey2016_foodstamps"]
   end
 
+  test 'post net promoter score survey results' do
+    sign_in @teacher
+
+    NPS_VALUE = 10
```

Comment:
I'm surprised that you don't get dynamic constant assignment here (this is a proc not a method)... but in any case you probably don't need this to be a constant.